server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/sources/coinpaprika/CoinPaprikaRateSource.java

```diff
@@ -76,6 +76,7 @@
         CRYPTOCURRENCIES.put(CryptoCurrency.POT.getCode(), "pot-potcoin");
         CRYPTOCURRENCIES.put(CryptoCurrency.REP.getCode(), "rep-augur");
         CRYPTOCURRENCIES.put(CryptoCurrency.SMART.getCode(), "smart-smartcash");
+        CRYPTOCURRENCIES.put(CryptoCurrency.SMART.getCode(), "smart-aquanow");
```

Comment:
Invalid. Cannot be there twice.

Reply:
Removed. We don't have any cryptocurrency.